server/utilis/validation/contactusValidation.js

```diff
@@ -3,7 +3,7 @@ const Joi = require('joi');
 const contactusValidation = Joi.object({
   message: Joi.string().required(),
   email: Joi.string().email().required(),
-  name: Joi.string().required(),
+  username: Joi.string().required(),
   mobile: Joi.number().required(),
```

Comment:
please, make sure that validation on the client-side and server-side are identical,
the message and phone number is different

validation on  server
```]
  mobile: yup.number().min(9, 'mobile must be at least 9 digits').required(),
  message: yup.string().max(200, 'message must be under 200 char').required(),
```